app/src/main/java/test1/daihocbachkhoa/tranphuc1995/assignment1/MainActivity.java

```diff
@@ -0,0 +1,51 @@
+package test1.daihocbachkhoa.tranphuc1995.assignment1;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+import java.util.ArrayList;
+
+public class MainActivity extends AppCompatActivity {
+
+    RecyclerView recyclerView;
+    RecyclerViewAdapter adapter;
+    ArrayList<Beverages> arlDrink;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        createData();
+        init();
+    }
+    public void init(){
+        recyclerView = (RecyclerView)findViewById(R.id.recyclerViewList);
+        adapter = new RecyclerViewAdapter(arlDrink);
+        LinearLayoutManager layoutManager = new LinearLayoutManager(getApplicationContext());
+        layoutManager.setOrientation(LinearLayoutManager.VERTICAL);
+        recyclerView.setLayoutManager(layoutManager);
+        recyclerView.setAdapter(adapter);
+
+    }
+
+    public void createData(){
```

Comment:
How about `createData` actually returns the mock data here

```
public List<Beverage> createData() {
....
```
We explicitly create mock data and assign to our global fields `arlDrink`

Remember: If you have to choose between explicit and implicit ways to do thing, prefer the `explicit` one.